src/network.js

```diff
@@ -14,6 +16,12 @@ const TX_BROADCAST_SERVICE_ZONE_FILE_ENDPOINT = 'zone-file'
 const TX_BROADCAST_SERVICE_REGISTRATION_ENDPOINT = 'registration'
 const TX_BROADCAST_SERVICE_TX_ENDPOINT = 'transaction'
 
+export function hash160(buff: Buffer) {
```

Comment:
This function is already defined in `operations/utils.js` -- we should probably just import it here.

Reply:
Fixed and added a unit test.